src/index.pug

```diff
@@ -100,7 +100,7 @@ block content
                     p Chat with us #[br] on Gitter
                 .col-md-3.col-sm-6.col-xs-12: a(href='https://discord.gg/mixer' target='_blank')
                     .slide-features-img
-                        img(src='/img/icon/chat-double-bubble-1.svg' alt='Go to our Dev Forums' target='_blank')
+                        img(src='/img/icon/chat-double-bubble-1.svg' alt='Talk to other developers on Discord' target='_blank')
```

Comment:
This is actually rly confusing to read, why does the `img` tag have it's own `target` attribute? Is the image not nested in the link?

Reply:
oh, yea, that's weord 😛

Reply:
Oh wut, never noticed this. I'll remove it.